lib/timeline/Timeline.js

```diff
@@ -301,12 +301,27 @@ Timeline.prototype.setData = function (data) {
  *                                    Default duration is 500 ms, and default easing
  *                                    function is 'easeInOutQuad'.
  *                                    Only applicable when option focus is true.
+ *                                `selectEvent: Event`
+ *                                    Call the select event manually to make sure the selected items
+ *                                    will be highlighted in the timeline. Specific use for JQuery-
```

Comment:
The comment is too specific for me. I see why we can use this addition, but the comment doesn't need the jQuery D&D mention.